src/Workspaces/CSharp/Portable/Execution/CSharpOptionsSerializationService.cs

```diff
@@ -32,6 +34,15 @@ public override void WriteTo(ParseOptions options, ObjectWriter writer, Cancella
             writer.WriteValue(options.PreprocessorSymbolNames.ToArray());
         }
 
+        public override void WriteTo(OptionSet options, ObjectWriter writer, CancellationToken cancellationToken)
+        {
+            WriteOptionSetTo(options, LanguageNames.CSharp, writer, cancellationToken);
+
+            WriteOptionTo(options, CSharpCodeStyleOptions.UseImplicitTypeForIntrinsicTypes, writer, cancellationToken);
```

Comment:
there is no way to do that. some option is defined in language specific dll. roslyn option can't be used without loading dll that has option definition.